heat/core/operations.py

```diff
@@ -172,8 +172,23 @@ def __local_op(operation, x, out, **kwargs):
         if kwargs_dtype:
             del kwargs['dtype']
         result = operation(x._DNDarray__array.type(torch_type), **kwargs)
-        return dndarray.DNDarray(result, tuple(result.shape), promoted_type if kwargs_dtype is None else kwargs_dtype,
-                                 x.split, x.device, x.comm)
+        split = x.split
```

Comment:
For what local operations would this be the case? The ones that I can think of at the moment are just element-wise operations. Everything else is either a binary operation and/or a reduction.

Reply:
Yes the target of this change are the element wise operations. It might be good to have them separated. This way no communication would be necessary because the shape stays the same